package.json

```diff
@@ -37,7 +37,7 @@
         "start": "http-server -c-1",
         "watch": "rollup --config --watch"
     },
-    "version": "0.2.0",
+    "version": "0.2.5",
```

Comment:
Why 0.2.5 and not 0.2.1? 🤔

Reply:
It felt right at the time, but now your suggestion makes more sense...

Reply:
updated both to 0.2.1